Flake8 error in settings/local.py

```
someproject/settings/local.py:47:1: E305 expected 2 blank lines after class or function definition, found 1
ERROR: InvocationError: '/home/tom/Projects/someproject/.tox/py35/bin/flake8 someproject apps'
```

--- a/{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings/local.py
+++ b/{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings/local.py
@@ -44,6 +44,7 @@ def show_toolbar(request):
 
     return os.environ.get('DEBUG_TOOLBAR', '1') == '1'
 
+
 INSTALLED_APPS += [
     'debug_toolbar',
 ]

